### ISSUE TITLE
TV series UI bug

### ISSUE THREAD

**Issue by kisst** [TorrentLookup/Flixtor#30]
I have to call it to bug, because it's really unpractical, popup in a popup, in a popup, that's madness.
If you search for the series, first of all the filter should stay, but there is already one ticket about that, but then when you open up one, then that should be an separated page, with already the list of the episodes, and once that is finished then the next page should be the series page and not the main page again.
So the user journey is bad as it is, and just as an side-note: in general the UI still not the best. Check out this screenshots, that's something what looks good ...
https://cloud.githubusercontent.com/assets/2908209/2728277/b9656952-c5ed-11e3-8682-438e2431b3fb.png

**Comment by gianpaj** [comment]
Where is that UI from?

**Comment by Japoilski** [comment]
I suppose he/she did it and  it looks good.
We will update the UI but for now we still need to address a lot more important issues.
Also, let's make another issue for the new UI we could implement.
